letsencrypt/cli.py

```diff
@@ -1139,6 +1142,27 @@ def main(cli_args=sys.argv[1:]):
 
     return args.func(args, config, plugins)
 
+def check_config_sanity(args):
+    """Validate command line options and display error message if
+    requirements are not met.
+
+    :param args: Command line options
+    :type args: :class:`argparse.Namespace`
+
+    """
+    # Domain checks
+    if args.domains is not None:
+        # Check if there's a wildcard domain
+        if any(d.startswith("*.") for d in args.domains):
+            raise errors.ConfigurationError("Error: Wildcard domains are not supported")
```

Comment:
nit: can we not prepend everything with an "Errror:"? :) If the end outcome output needs to be fixed, then that's not the right place.